spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -36,8 +68,13 @@
 
   def default_params
     {
-      envelope: { to: 'Some Identifier <some-identifier@example.com>', from: 'Joe User <joeuser@example.com>' },
-      headers: { Subject: 'Re: [ThisApp] That thing', Cc: 'emily@example.com' },
+      envelope: { to: 'some-identifier@example.com', from: 'joeuser@example.com' },
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.